deps/blas.mk

```diff
@@ -46,7 +46,7 @@ $(BUILDDIR)/$(OPENBLAS_SRC_DIR)/build-compiled: | $(BUILDDIR)/objconv/build-comp
 endif
 endif
 
-OPENBLAS_FFLAGS :=
+OPENBLAS_FFLAGS := $(FFLAGS) $(JFFLAGS)
 OPENBLAS_CFLAGS :=
```

Comment:
Could this request perhaps be moved to an issue. Is it necessary to solve before merging this PR?